src/elements/content-explorer/ItemGrid.js

```diff
@@ -21,13 +22,14 @@ const ItemGrid = ({ currentCollection, rootId, ...rest }: Props) => {
     const slotRenderer = (slotIndex: number): ?React.Element<any> => {
         const item: ?BoxItem = getProp(currentCollection, `items[${slotIndex}]`);
 
-        return item ? <ItemGridCell item={item} rootId={rootId} {...rest} /> : null;
+        return item ? <ItemGridCell item={item} onItemSelect={onItemSelect} rootId={rootId} {...rest} /> : null;
```

Comment:
Does `onItemSelect` need to be defined explicitly or would it be passed in as part of `...rest`?